cpp/src/IceGrid/Scanner.l

```diff
@@ -36,26 +22,21 @@ using namespace IceGrid;
 #      undef yywrap
 #      define yywrap() 1
 #   endif
-#   define YY_NO_UNISTD_H
 #endif
 
 #ifdef __SUNPRO_CC
 #   ifdef yywrap
 #      undef yywrap
 #      define yywrap() 1
 #   endif
```

Comment:
We can probably remove support for the the Sun compiler.

Reply:
Alright, sounds good to me! Consider it removed!

Reply:
Actually, I think I'll make a separate PR after this one, we'll I'll prune it from everything, since we still use it in lots of other places too...

Reply:
Yes, let's remove all old compilers and OSs in a separate PR.